app/submitter/converter.py

```diff
@@ -47,6 +47,11 @@ class SubmitterConstants(object):
 
 class Converter(object):
 
+    """
+    Date Format expected by SDX
+    """
+    SDX_DATE_FORMAT = "%d/%m/%Y"
```

Comment:
I think this should be configurable, can we move it to settings.py and expose as an environment variable?